src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,39 @@
 import React from 'react';
+import { Comment } from '../../types/comment';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
-);
+type Props = {
+  commentInfo: Comment;
+};
+
+export const CommentInfo: React.FC<Props> = (props) => {
+  const {
+    commentInfo,
+  } = props;
```

Comment:
Do this in params
fix in all cases

```suggestion
export const CommentInfo: React.FC<Props> = ({ comentInfo }) => {
```